spring-boot-project/spring-boot-actuator-autoconfigure/src/main/asciidoc/endpoints/integrationgraph.adoc

```diff
@@ -0,0 +1,40 @@
+[[integrationgraph]]
+= Spring Integration graph (`integrationgraph`)
+
+The `integrationgraph` endpoint exposes a graph containing all Spring Integration components.
+
+
+
+[[integrationgraph-retrieving]]
+== Retrieving the Spring Integration graph
+
+To retrieve the information about the application, make a `GET` request to
+`/actuator/integrationgraph`, as shown in the following curl-based example:
+
+include::{snippets}integrationgraph/graph/curl-request.adoc[]
+
+The resulting response is similar to the following:
+
+include::{snippets}integrationgraph/graph/http-response.adoc[]
+
+
+
+[[integrationgraph-retrieving-response-structure]]
+=== Response Structure
+
+The response contains all Spring Integration components used within the application, as well as the links between them.
+More info about the structure can be found in the https://docs.spring.io/spring-integration/reference/htmlsingle/#integration-graph[subchapter] for the integration graph in the reference documentation of Spring Integration.
```

Comment:
I wouldn't use `htmlsingle` here as a cross-link. Consider to switch to this one: https://docs.spring.io/spring-integration/reference/html/system-management-chapter.html#integration-graph.

Also I don't think that `info` is formal word. The `information` doesn't pull any questions from me :smile: